docs/installation.rst

```diff
@@ -38,13 +38,14 @@ OpenSSL releases:
 
 .. warning::
     OpenSSL 1.0.0 is no longer supported by the OpenSSL project. Cryptography
-    will drop support for it in the next release.
+    will drop support for it in a future release.
```

Comment:
This line needs to be changed back to `the next` -- looks like a merge resolution mistake.